microsoft-365/admin/manage/manage-plugins-for-copilot-in-integrated-apps.md

```diff
@@ -77,6 +77,9 @@ Admins can deploy or remove extensions for Copilot across the whole organization
 
 Admins can assign or unassign extensions for Copilot to specific users or groups by using the same gestures and controls that work for any other app in the Microsoft 365 admin center. Admins can select an app that has a plugin for Copilot from the Available apps or Deployed apps list and click on the **Assign** or **Unassign** button. Admins can also choose to assign or unassign the app to everyone or specific users or groups. Assigning or unassigning an app affects its availability and functionality in Copilot as well as other host products, such as Outlook, Teams, or Microsoft 365.
 
+> [!IMPORTANT]
+> Pre-approved or pre-installed apps by admins are usable in Microsoft Copilot when the apps are updated to support Microsoft Copilot.
```

Comment:
A few queries.

* Does this apply to pre-approved or pre-installed apps only? Are other apps (Blocked today but approved tomorrow) not available when updated?
* What does it mean for an app to be "usable in Copilot"? Are we trying to say that apps are available to users OR are apps auto-updated to use Copilot?
* What does it mean for an app to be updated "to support Copilot"? Should we say apps that use AI? Should we break this into 2 sentences and frontload the fact that ISVs can update their existing app to use AI and the updated app is a smart app?
* Somewhere we have a high-level doc to show how AI apps are created and it showcases the various M365 and Azure offering used in the AI apps. Would it make sense to link to this overview, developer article from the "updated to support Copilot" text?
* Create links to how admins pre-approve and how admins pre-install an app.